test/core/simd/meta/simd_float_op.py

```diff
@@ -160,28 +167,40 @@ def binary_op(self, op: str, p1: str, p2: str) -> str:
         if op == 'min':
             if '-0x0p+0' in [p1, p2] and '0x0p+0' in [p1, p2]:
                 return '-0x0p+0'
-            result = min(f1, f2)
+            if hex_form:
+                result = min(f1, f2)
```

Comment:
This logic would be easier to follow if this was simply `return min(f1, f2).hex()`. Same below.

Reply:
Indeed, done.